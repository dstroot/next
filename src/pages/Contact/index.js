import React, { useEffect } from 'react';
// component
import SectionHereToHelp from '../../components/Sections/SectionHero/SectionHereToHelp';
import SectionAskUs from '../../components/Sections/SectionTwoColumns/SectionAskUs';
import Footer from '../../components/Footer';

const Contact = () => {
  // Set the page title and position using the useEffect hook
  useEffect(() => {
    document.title = `NEXT â€¢ Contact`;
    window.scrollTo(0, 0);
  });

  return (
    <div>
      <SectionHereToHelp />
      <SectionAskUs />
      <Footer />
    </div>
  );
};

export default Contact;
