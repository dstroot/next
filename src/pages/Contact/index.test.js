import React from 'react';
import { create } from 'react-test-renderer';
import Contact from '../Contact';

describe('Contact', () => {
  it('it should render', () => {
    let tree = create(<Contact />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
