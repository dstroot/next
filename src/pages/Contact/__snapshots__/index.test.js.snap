// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact it should render 1`] = `
<div>
  <img
    alt="Two Lights"
    src="twolights.svg"
    style={
      Object {
        "left": "0",
        "marginLeft": "auto",
        "marginRight": "auto",
        "marginTop": "12vh",
        "minWidth": "50px",
        "position": "absolute",
        "right": "0",
        "width": "42vh",
      }
    }
  />
  <div
    className="background"
    style={
      Object {
        "backgroundImage": "url(contactladys.svg)",
      }
    }
  >
    <div
      className="divText"
    >
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col text-center"
          >
            <h1
              className="nxt_heading-xlarge"
            >
              Here to help whenever you need us!
            </h1>
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col-md-8 offset-md-2 text-center"
          >
            <p
              className="mt-4 circular"
            >
              Insurance products to complement your life goals and passions. We help you go after whatever's next for you.
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="py-6 bg-light"
  >
    <div
      className="container"
    >
      <h1
        className="react-reveal text-center nxt_heading-xlarge undefined"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        Ask us anything
      </h1>
      <div
        className="row"
      >
        <div
          className="react-reveal col-md-5 order-2 mt-5 text-left"
          style={
            Object {
              "opacity": undefined,
            }
          }
        >
          <div>
            <h2>
              Next by Pacific Life
            </h2>
            <a
              href="mailto: hello@meetnext.com"
            >
              hello@meetnext.com
            </a>
            <p>
              (833) 646-NEXT (6398) 
            </p>
          </div>
        </div>
        <div
          className="col-md-7 order-1 text-left"
        >
          <div
            className="mt-5"
          >
            <div
              className="react-reveal"
              style={
                Object {
                  "opacity": undefined,
                }
              }
            >
              <div
                className="mb-6 text-left nxt_body-xsmall"
              >
                <form
                  onSubmit={[Function]}
                >
                  <div>
                    <label>
                      Name
                    </label>
                    <input
                      className="form-control mb-3"
                      name="name"
                      onChange={[Function]}
                      required={true}
                      type="text"
                      value=""
                    />
                  </div>
                  <div>
                    <label>
                      Email
                    </label>
                    <input
                      className="form-control mb-3"
                      name="email"
                      onChange={[Function]}
                      required={true}
                      type="email"
                      value=""
                    />
                  </div>
                  <div>
                    <label>
                      Phone
                    </label>
                    <input
                      autoComplete="tel"
                      className="form-control mb-3"
                      maxLength="14"
                      name="phone"
                      onBlur={[Function]}
                      onChange={[Function]}
                      type="tel"
                      value=""
                    />
                  </div>
                  <div>
                    <label>
                      Message
                    </label>
                    <textarea
                      className="form-control mb-3"
                      name="message"
                      onChange={[Function]}
                      required={true}
                      type="text"
                      value=""
                    />
                  </div>
                  <button
                    className="btn btn-primary rounded-pill nxt_body-small px-4 py-2 submit"
                  >
                    Submit
                  </button>
                </form>
                <h4
                  className="alert"
                >
                  We received your message! 
                  <br />
                   We’ll get back to you shortly.
                </h4>
              </div>
            </div>
            <div>
              <div
                className="mt-4 pr-5"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="container-fluid bg-dark"
  >
    <div
      className="row"
    >
      <div
        className="col p-3"
      >
        <img
          alt="Logo"
          className="pr-3 pb-4"
          src="NXT_logo_footer_dark.svg"
        />
        <span
          className="nxt_footer"
        >
          By using this website you accept our
           
          <a
            href="/terms"
            onClick={[Function]}
          >
            <span
              className="text-accent"
            >
              Terms of Use
            </span>
          </a>
          ,
           
          <a
            href="/social"
            onClick={[Function]}
          >
            <span
              className="text-accent"
            >
              Social Media Guidelines
            </span>
          </a>
          , and
           
          <a
            href="/privacy"
            onClick={[Function]}
          >
            <span
              className="text-accent"
            >
              Privacy Policy
            </span>
          </a>
        </span>
        <small>
          <div
            className="nxt_footer"
          >
            <p>
              No bank guarantee • Not a deposit • May lose value • Not FDIC/NCUA insured • Not insured by any federal government agency
            </p>
            <p>
              Pacific Life, its affiliates, its distributors, and respective representatives do not provide tax, accounting, or legal advice. Any taxpayer should seek advice based on the taxpayer's particular circumstances from an independent tax advisor or attorney.
            </p>
            <p>
              <i>
                Pacific Life is a product provider. It is not a fiduciary and therefore does not give advice or make recommendations regarding insurance or investment products.
              </i>
            </p>
            <p>
              Pacific Life Insurance Company (Newport Beach, CA) is licensed to issue insurance products in all states except New York. Product availability and features may vary by state. Fixed annuity products are available through licensed third parties.
            </p>
            <p>
              © 
              2019
               Pacific Life Insurance Company, Newport Beach, CA.
            </p>
          </div>
        </small>
      </div>
    </div>
  </div>
</div>
`;
